plugins/source/okta/client/spec_test.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/cloudquery/codegen/jsonschema"
+)
+
+func TestJSONSchema(t *testing.T) {
+	jsonschema.TestJSONSchema(t, JSONSchema, []jsonschema.TestCase{
+		{
+			Name: "empty spec",
+			Spec: `{}`,
+			Err:  true,
+		},
+		{
+			Name: "spec with token",
+			Spec: `{"token": "tok"}`,
+			Err:  true,
+		},
+		{
+			Name: "spec with domain",
+			Spec: `{"domain": "https://domain.okta.com"}`,
+		},
+		{
+			Name: "spec with token and domain",
+			Spec: `{"token": "tok", "domain": "https://domain.okta.com"}`,
+		},
+		{
+			Name: "spec with token and invalid domain",
+			Spec: `{"token": "tok", "domain": "https://<CHANGE_THIS_TO_YOUR_OKTA_DOMAIN>.okta.com"}`,
+			Err:  true,
+		},
+		{
+			Name: "spec with token and domain and empty rate limit",
+			Spec: `{"token": "tok", "domain": "https://domain.okta.com", "rate_limit": {}}`,
+		},
+		{
+			Name: "spec with token and domain and null rate limit",
+			Spec: `{"token": "tok", "domain": "https://domain.okta.com", "rate_limit": null}`,
+		},
+		{
+			Name: "spec with token and domain and rate limit",
+			Spec: `{"token": "tok", "domain": "https://domain.okta.com", "rate_limit": {"max_backoff": "60s"}}`,
+		},
+		{
+			Name: "spec with token and domain and invalid rate limit",
+			Spec: `{"token": "tok", "domain": "https://domain.okta.com", "rate_limit": {"max_backoff": true}}`,
```

Comment:
remove token from tests (only leave for the tests that are actually testing token)

Reply:
Or merge https://github.com/cloudquery/cloudquery/pull/16498 & reintroduce requirement for `token` (better IMO)